src/main/java/org/traccar/model/Command.java

```diff
@@ -72,6 +71,10 @@ public class Command extends BaseCommand {
 
     public static final String KEY_UNIQUE_ID = "uniqueId";
     public static final String KEY_FREQUENCY = "frequency";
+
+    public static final String KEY_ENGINE_OFF_INTERVAL = "engine_off_interval";
+    public static final String KEY_ENGINE_ON_INTERVAL = "engine_on_interval";
+    public static final String KEY_HEARTBEAT_INTERVAL = "heartbeat_interval";
```

Comment:
We don't add new commands just like this. We only add if something is used by at least a few protocol and is generic enough.

Reply:
Used KEY_FREQUENCY to get the params